src/common/hooks/use-edit-postconditions.ts

```diff
@@ -0,0 +1,63 @@
+import { useSetRecoilState } from 'recoil';
+import { currentPostConditionIndexStore, postConditionsStore } from '@store/transaction';
+import { Asset, selectedAssetStore } from '@store/asset-search';
+import { useFetchBalances } from '@common/hooks/use-account-info';
+import { useDoChangeScreen } from '@common/hooks/use-do-change-screen';
+import { useCallback } from 'react';
+import { addressToString, PostCondition, PostConditionType } from '@stacks/transactions';
+import { toHumanReadableStx, truncateMiddle } from '@stacks/ui-utils';
+import { ScreenPaths } from '@store/types';
+
+function useEditPostcondition({ pc, index }: { pc: PostCondition; index: number }) {
```

Comment:
```suggestion
function useEditPostCondition({ pc, index }: { pc: PostCondition; index: number }) {
```
Per this and the file names, I think we could consider post condition two words. Edit, just seen fara's comment of the same